ExampleCheckout/Sources/Swift/ViewController.swift

```diff
@@ -131,16 +131,13 @@ extension ViewController: ChargePresetDelegate {
 }
 
 extension ViewController {
-    var tintColor: UIColor {
-        get { view.tintColor }
-        set {
-            view.tintColor = newValue
-            themeSwitch.onTintColor = newValue
-            textField.tintColor = newValue
-            showPaymentListButton.backgroundColor = newValue
-            chargePresetAccountButton.backgroundColor = newValue
-            setNavigationBarColor(to: newValue)
-        }
+    fileprivate func setTintColor(to color: UIColor) {
```

Comment:
`private` is enough, no?

```suggestion
    private func setTintColor(to color: UIColor) {
```